src/Extension.ml

```diff
@@ -28,6 +28,37 @@ module Server = struct
     { command = Path.toString command; args; options = { env = Process.env } }
 end
 
+module TraceSetting = struct
```

Comment:
As mentioned earlier, I think this option is already handled by the client implementation. So we don't actually need to do any of this.